spec/ffprober/parser_spec.rb

```diff
@@ -26,6 +26,24 @@ def shared_specs
     end
   end
 
+    describe 'subtitle_streams' do
+    it 'should determine the correct duration_ts of the first subtitle stream' do
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.